Framework/API/src/MultiPeriodGroupWorker.cpp

```diff
@@ -280,15 +280,15 @@ void MultiPeriodGroupWorker::validateMultiPeriodGroupInputs(
                 currentGroup->getItem(j));
         Property *nPeriodsProperty =
             currentNestedWS->run().getLogData("nperiods");
-        size_t nPeriods = atoi(nPeriodsProperty->value().c_str());
+        size_t nPeriods = std::stoi(nPeriodsProperty->value());
         if (nPeriods != benchMarkGroupSize) {
```

Comment:
Would [std::stoul](http://en.cppreference.com/w/cpp/string/basic_string/stoul) be more precise?